prow/cmd/sub/README.md

```diff
@@ -75,3 +76,53 @@ This will find and start the periodic job ```my-periodic-job```, and add / overw
 annotations and envs to the Prow job. The ```prow.k8s.io/pubsub.*``` annotations are
 used to publish job status.
 
+#### Presubmit Prow Jobs
+
+Triggering presubmit job is similar to periodic jobs, requires setting `refs` instructing
+presubmit jobs how to clone source code.
+
+```json
+{
+  // Common fields as above
+  "name":"my-presubmit-job",
+  "envs":{...},
+  "labels":{...},
+  "annotations":{...},
+
+  "refs":{
+    "base_ref": "main",
+    "base_sha": "abc123",
+    "pulls": [
+      {
+        "sha": "def456"
```

Comment:
We should link to the full `Refs` struct since there are a lot of additional fields that may be needed to achieve expected behavior: https://github.com/kubernetes/test-infra/blob/62f15287bd0c21ea8018bd4e8b6958d0f05177b4/prow/apis/prowjobs/v1/types.go#L882-L923

e.g. `CloneURI`, `PathAlias`, Pull `Number` and `Author`.

There are also two required fields that we're missing (and for which we are also missing validation in Sub): `Org` and `Repo` :man_facepalming:

Reply:
🤦

Reply:
Done